Задача 2. Проведите тест гипотезы. Утверждается, что шарики для подшипников, изготовленные
автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из
n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

In [4]:
from math import *

In [5]:
M = 17
D = 4
a = 0.05
n = 100
mx = 17.5

Так как известна дисперсия, используем Z –критерий.

In [6]:
z = (mx - M) / (sqrt(D) / sqrt(n))
print('Z = ', z)

Z =  2.5


Нулевая гипотеза утверждает,что между средними диаметрами в генеральной совокупности и в выборке нет статистически значимых отклонений.
Альтернативная гипотеза утверждает, что средний диаметр шариков из выборки больше (есть статистически значимые отклонения) среднего диаметра шариков из генеральной совокупности.

для расчётного Z = 2.5 P-value = 1 - 0,9938
для α = 0,05 p = 1 - α, табличное Z = 1.645
т.к. Z расчётное > Z табличного (P-value < α) принимаем альтернативную гипотезу (между средними диаметрами в генеральной совокупности и в выборке есть статистически значимые отклонения)